rest-api-spec/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
   clusterModules project(":modules:mapper-extras")
   clusterModules project(":modules:rest-root")
   clusterModules project(":modules:reindex")
+  clusterModules project(':modules:analysis-common')
```

Comment:
Add `analysis-common` to REST tests so synonyms analyzers are available for tests